crates/core/src/client.rs

```diff
@@ -69,11 +69,11 @@ impl Default for NodeData {
 pub struct BeerusClient {
     /// Ethereum light client
     #[cfg(not(target_arch = "wasm32"))]
-    pub helios_client: Arc<Client<FileDB>>,
+    helios_client: Arc<Client<FileDB>>,
     #[cfg(target_arch = "wasm32")]
-    pub helios_client: Arc<Client<ConfigDB>>,
+    helios_client: Arc<Client<ConfigDB>>,
     /// StarkNet light client
-    pub starknet_client: JsonRpcClient<HttpTransport>,
+    pub(crate) starknet_client: JsonRpcClient<HttpTransport>,
```

Comment:
Is this just for external clarity?

Reply:
as for now, beerus protect calling with more fresh block n then beerus local, so exposing client would be not consistent, as far as i understand